lib/lotus/cli.rb

```diff
@@ -3,6 +3,29 @@
 require 'lotus/version'
 
 module Lotus
+
+  class DBSubcommand < Thor
```

Comment:
maybe we should move this class to a file, because there will be more subcommands, what do you think?